extensions/SemanticMediaWiki/DefaultSettings.php

```diff
@@ -456,7 +456,7 @@
 # the local source (which in turn cannot be disabled or set explicitly).
 # Set a new store like this: $smwgQuerySources['freebase'] = "SMWFreebaseStore";
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this commented out code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS125)